sos/policies/arch.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
What's this for?

Reply:
It's unused, and has been removed. (Incidentally, `pyflakes` shows 17 other instances of unused imports under `../sos`.)